snprc_ehr/src/client/SsrsReporting/SsrsReporting.tsx

```diff
@@ -6,7 +6,7 @@ import "./styles/ssrsReporting.scss";
 import { Button } from "react-bootstrap";
 import ReportSelectionPanel from "./components/ReportSelectionPanel";
 import InfoPanel from "../Shared/components/InfoPanel";
-import { OptionsType } from "react-select";
+import { OptionsType } from 'react-select';
```

Comment:
necessary change?

Reply:
Nope, I ended up changing the version number of @labkey/components back and forth a few times and had to change this OptionsType in the process so I must have manually added this line back in.